CodeOfConduct.md

```diff
@@ -0,0 +1,69 @@
+# Seattle CoffeeOps - Code Of Conduct
+
+## ANTI-HARASSMENT POLICY
+
+CoffeeOps is dedicated to providing a harassment-free
+experience for everyone, regardless of gender, sexual orientation,
+disability, physical appearance, body size, race, or religion. We do not
+tolerate harassment of event participants in any form. Sexual language and
+imagery is not appropriate for any event venue.
+Participants violating these rules may be sanctioned or expelled
+from the event at the discretion of the [organizers](https://www.meetup.com/Seattle-CoffeeOps/members/?op=leaders).
+
+Harassment includes offensive verbal comments related to gender, sexual
+orientation, disability, physical appearance, body size, race, religion, sexual
+images in public spaces, deliberate intimidation, stalking, following,
+harassing photography or recording, sustained disruption of talks or other
+events, inappropriate physical contact, and unwelcome sexual attention.
+Participants asked to stop any harassing behavior are expected to comply
+immediately.
+
+If a participant engages in harassing behavior, the event organizers may
+take any action they deem appropriate, including warning the offender or
+expulsion from the group.
+
+If you are being harassed, notice that someone else is being harassed, or have
+any other concerns, please contact a member of the leadership team immediately.
+
+We expect participants to adhere to the code of conduct at all event venues and
+CoffeeOps-related social events.
+
+## CODE OF CONDUCT
+
+1. I am an attendee at CoffeeOps, learning from and sharing
+with other CoffeeOps attendees in an effort to better myself and my industry. I
+co-create the experience with fellow attendees. I am prepared to give my
+energy, presence and sensitivity to creating the best possible experience for
+myself and others.
+
+1. I am coming to CoffeeOps to interact with people. I understand that imagery
+and language which is suggestive or derogatory will offend and make people
+uncomfortable. I also understand that people may have boundaries and
+sensibilities different from my own. I will accept without question when
+informed that something is offensive or unacceptable in the context of the
+CoffeeOps event.
+
+1. I will never intentionally harass or offend another attendee regardless of
+gender, sexual orientation, disability, appearance, size, race or religion and
```

Comment:
Grammar nit: I'd add a semicolon after religion, and change the `and` to an `I` so it reads: `race or religion; I will not abide`.